packages/agw-client/src/eip5792.ts

```diff
@@ -0,0 +1,38 @@
+import type { Address, Hex } from 'viem';
+
+interface Capability {
+  supported: boolean;
+}
+
+interface ChainCapabilities {
+  paymasterService?: Capability;
+  sessionKeys?: Capability;
+  atomicBatch?: Capability;
+}
+
+type WalletCapabilities = Record<`0x${string}`, ChainCapabilities>;
+
+export interface SendCallsParams {
+  version: string;
+  from: Address;
+  calls: {
+    to?: Address | undefined;
+    data?: Hex | undefined;
+    value?: Hex | undefined;
+    chainId?: Hex | undefined;
+  }[];
+  capabilities?: WalletCapabilities | undefined;
+}
+
+export const agwCapablities: WalletCapabilities = {
```

Comment:
Are there any other capabilities that we support? Maybe `paymasterService`?

Reply:
The paymaster capability in wallet_sendCalls takes a URL, so before supporting this we will need to figure out how to handle this. For standard 4337 this is how paymasters work but for our paymasters it's just encoded onchain data so this isn't entirely to the spec.